package.json

```diff
@@ -46,7 +46,7 @@
     "copy-webpack-plugin": "4.0.1",
     "coveralls": "^2.11.15",
     "css-loader": "^0.26.0",
-    "d3": "^4.4.1",
+    "d3": "^4.9.1",
```

Comment:
If we need to use multiple d3 versions, it's possible, even relatively easy with webpacker - all the code which needs that specific version needs to live in a pack that gets compiled together with the right d3 version, and use a d3 version which can be `require`d but does not create a global object (looks like https://www.npmjs.com/package/d3-no-global-export would do the trick).